src/components/ContactsPage.js

```diff
@@ -73,6 +89,15 @@ class ContactsPage extends React.Component {
           <div className="ui two column stackable center aligned grid">
             <div className="column">
               {this.renderContactsList()}
+              <div className="ui segment">
+                <button
+                  className="fluid ui button olive"
+                  type="button"
+                  onClick={this.deleteSelectedContacts}
+                  disabled={this.isContactSelected()}
```

Comment:
Rename this to `this.anyContactSelected` and use the negation of it here: `!this.anyContactSelected `. Because reading in the it is now is not that clear. Sounds more like: 'disable this button if there isContactSelected'. And this is the otherway around: 'disable this if NO contacts selected'. And remove the negation inside the `isContactSelected` function